test/rulesets/LinearAlgebra/dense.jl

```diff
@@ -119,6 +119,12 @@
                 test_rrule(f, B)
             end
         end
+        @testset "$f(unitary matrix)" begin
+            B = randn(ComplexF64, 4, 4)
+            U = exp(1im * (B + B'))
```

Comment:
I still prefer this simpler expression:
```suggestion
            U = exp(B - B')
```